cmake/utils.cmake

```
@@ -323,29 +323,34 @@ function(momo_strip_target target)
     return()
   endif()
 
-  if(NOT MSVC)
-    # TODO: detect LLVM IR bitcode and abort
+  if(MSVC)
```

maybe also return if MOMO_ENABLE_SANITIZER is set, to retain stack traces for sanitizer builds.
```suggestion
  if(MSVC OR MOMO_ENABLE_SANITIZER)
```